doc/overview.qbk

```diff
@@ -109,6 +139,6 @@ Asio library and the ideas upon which Beast is built.
 Beast would not be possible without the considerable time and patience
 contributed by David Schwartz, Edward Hennis, Howard Hinnant, Miguel Portilla,
 Nikolaos Bougalis, Scott Determan, Scott Schurr, and Ripple Labs for
```

Comment:
"and Ripple Labs for supporting its early development" doesn't fit when read out loud. Consider:

> Beast would not be possible without the considerable time and patience of David Schwartz, Edward Hennis, Howard Hinnant, Miguel Portilla, Nikolaos Bougalis, Scott Determan and Scott Schurr, or the support of [Ripple Labs](https://ripple.com) during its early development.

Also, thanks for the shout-out! Glad to be involved. :heart: Beast!